server-spi/src/main/java/org/keycloak/storage/client/ClientLookupProvider.java

```diff
@@ -63,6 +61,8 @@ public interface ClientLookupProvider {
 
     Stream<ClientModel> searchClientsByAttributes(RealmModel realm, Map<String, String> attributes, Integer firstResult, Integer maxResults);
 
+    Stream<ClientModel> searchClientsByAuthenticationFlowBindingOverrides(RealmModel realm, Map<String, String> overrides, Integer firstResult, Integer maxResults);
+
```

Comment:
As this is a potentially breaking change to a public SPI, this method should have a default implementation, shouldn't it?